features/yield/YieldCalculator.tsx

```diff
@@ -58,8 +60,7 @@ const YieldCalculator = () => {
 
     // `yieldPerUnit` = (FACE/yUSD_PX)^(1/(365/DAYS_TO_EXP)) - 1,
     // where FACE = $1. More details: https://www.bankrate.com/glossary/a/apy-annual-percentage-yield/
-    const yieldPerUnit =
-      Math.pow(1 / _tokenPrice, 1 / (_daysToExpiry / DAYS_PER_YEAR)) - 1;
+    const yieldPerUnit = (1 - _tokenPrice) * (DAYS_PER_YEAR / _daysToExpiry);
```

Comment:
@chrismaree @hlambur can you take a quick look at this? Chris thinks that we are calculating APY incorrectly